dash/development/_r_components_generation.py

```diff
@@ -374,25 +384,40 @@ def write_help_file(name, props, description, prefix):
     file_path = os.path.join('man', file_name)
     with open(file_path, 'w') as f:
         f.write(help_string.format(
-            funcname=format_fn_name(prefix, name),
+            funcname=funcname,
             name=name,
             default_argtext=textwrap.fill(default_argtext,
                                           width=80,
                                           break_long_words=False),
             item_text=item_text,
             description=description.replace('\n', ' ')
         ))
-
-
-def write_class_file(name, props, description, project_shortname, prefix=None):
+    if rpkg_data is not None and rpkg_data.get('r_examples') is not None:
+        ex = rpkg_data.get('r_examples')
+        the_ex = ([e for e in ex if e.get("name") == funcname] or [None])[0]
+        result = ""
+        if the_ex and "code" in the_ex.keys():
+            result += wrap("examples",
+                           wrap("dontrun" if the_ex["dontrun"] else "",
```

Comment:
`if the_ex.get("dontrun")`
When would a user want or not want `dontrun`?

Reply:
There's a slight delay when executing inline examples within R packages; CRAN requires that each example run and exit without errors in a second or two. Dash apps aren't typically terminated until the user interrupts them; this is problematic for CRAN (and devtools) checks.

In addition, CRAN checks will fail if a dependency does not exist in the repository when examples are executed. Setting `\dontrun` to `TRUE` allows the package author to include examples but specify that they not be executed during package tests.

Reply:
My apologies for not catching the initial comment previously, change has been made!

Reply:
fixed in cf0d476

Reply:
no worries, the way I wrote it did look like I was quoting you rather than suggesting a change.